rpc/types_transaction_receipt.go

```diff
@@ -28,6 +28,15 @@ type MsgFromL1 struct {
 	Payload []*felt.Felt `json:"payload"`
 }
 
+type MessageStatusResp struct {
+	// The hash of a Starknet transaction
```

Comment:
nit:The hash of an l1 handler transaction

Reply:
Thanks! Done